safe-global/web-core
# fix: Dashboard widgets do not align correctly

## What it solves
This PR fixes the issue: fixes #1863 Dashboard widgets do not align correctly. Previously The "Connect & transact" widgets do not always stack/span the height of the "New in Safe" widget. This happened because **FeatureApps** Component was not able to take the full height of its parent component.
Resolves #

## How this PR fixes it
In this PR I gave 100% height to child component so that fixed the issue for medium sized screens. 
For large Sized screens the issue was with the min-width of the component. Due to this components were getting stacked horizontally. So increasing min-width to 500px for larger screen size solved the issue

## How to test it
Open the Safe on Gnosis Chain. Observe the "Connect & transact" widgets 
Connect & transact widget will span properly on larger resolutions and  span with the height of the "New in Safe" on medium resolutions,
## Screenshots

For Medium Sized Screens
![image](https://user-images.githubusercontent.com/32263182/231824471-363086ce-4809-4d02-9a4f-eef78b30faad.png)

For large Sized Screens
![image](https://user-images.githubusercontent.com/32263182/231824642-5b804833-eddc-4972-89fc-bd101e67cc29.png)


## Checklist
